river/TabletTool.zig

```diff
@@ -259,6 +259,8 @@ fn passthrough(tool: *TabletTool, tablet: *Tablet) void {
             tool.wp_tool.notifyMotion(result.sx, result.sy);
             return;
         }
+    } else {
+        tool.wlr_cursor.setXcursor(tablet.device.seat.cursor.xcursor_manager, "pencil");
```

Comment:
What's up with this change? It's not mentioned in the commit message...

Reply:
Merely changing the cursor to `"pencil"` on line 189 does not have the desired result: When a "new" tablet tool appears, i.e. the user moves the pen towards the pad, the cursor will be correct, however the `else` branch is needed because otherwise moving the tablet tool off a surface will not update the cursor image. So if a client sets it to something custom, that would persist until either the tablet tool disappears and re-appears or until the client or another client sets a different cursor image, which IMO is not the desired behaviour.

Reply:
Yeah that makes sense, I think I'd consider that a bugfix actually.